web/pgadmin/static/js/components/ReactCodeMirror/components/Editor.jsx

```diff
@@ -158,9 +158,8 @@ const defaultExtensions = [
   autoCompleteCompartment.of([]),
   EditorView.clipboardOutputFilter.of((text, state)=>{
     const lineSep = state.facet(eol);
-    // Fetch the primary selection from the editor's current state.
-    const selection = state.selection.main;
-    return state.doc.sliceString(selection.from, selection.to, lineSep);
+    // Fetch the selection range from the editor's current state.
+    return state.selection.ranges.map((range)=>state.sliceDoc(range.from, range.to)).join(lineSep) ?? '';
```

Comment:
did you check what sliceString does? We don't want to have any performance troubles.